### PR TITLE
Add export as CSV option for user data

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony ready for review :)

File: molo/profiles/admin.py [hunk 1]
```diff
@@ -26,3 +47,6 @@ def _mobile_number(self, obj, *args, **kwargs):
         if hasattr(obj, 'profile') and obj.profile.mobile_number:
             return obj.profile.mobile_number
         return ''
+
+    def _date_joined(self, obj, *args, **kwargs):
+        return obj.date_joined.date()
```
Comment (miltontony) [inline molo/profiles/admin.py]: Is this because you don't want to show the time

Comment (smarzban) [inline molo/profiles/admin.py]: yes :) is it necessarily to show the time?

Comment (miltontony) [inline molo/profiles/admin.py]: Yes, let's keep the time. It'll help when you need to find out whether the user joined in the morning or the evening.

**Comment by smarzban** [comment]
@miltontony ready for re-review :)

**Comment by miltontony** [comment]
:+1:

**Pull request merged**